book/content/ethics/ethics.md

```diff
@@ -14,7 +14,7 @@
 > why we think you should read the whole thing
 
 ## Chapter content
-> depending on the content, this might be more structured, e.g. with exercises, gotcha sections etc
+> depending on the content, this might be more structured, for example, with exercises, and gotcha sections
```

Comment:
```suggestion
```

I think quoted sentences describing what the chapter sections are just guidance for people writing the chapters. I think they should be removed so they are not rendered in the book.